cdc/processor/pipeline/sink.go

```diff
@@ -131,22 +134,24 @@ func (n *sinkNode) flushSink(ctx pipeline.NodeContext, resolvedTs model.Ts) (err
 		resolvedTs = n.targetTs
 	}
 	if resolvedTs <= n.checkpointTs {
-		return nil
+		return ok, nil
```

Comment:
I think we use `true/false` directly if more readable. I don't think it's make sense to define a boolean `ok` and return `ok/!ok`.

Reply:
Reasonable suggestion, thanks~